File: main/java/ru/carwash/models/Order.java
package ru.carwash.models;

import java.util.ArrayList;

/* Заказ */
public class Order {

    private CarWash carWash;
    private Car car;
    private String data; // TODO
    private String time; // TODO
    private ArrayList<Service> services; // список услуг в заказе
    private double price;
    private int status;

    public static int CANCELED_STATUS = 0;
    public static int ACCEPTED_STATUS = 1;
    public static int COMPLETED_STATUS = 2;

    public Order(int status) {
        this.status = status;
    }

    public CarWash getCarWash() {
        return carWash;
    }

    public void setCarWash(CarWash carWash) {
        this.carWash = carWash;
    }

    public Car getCar() {
        return car;
    }

    public void setCar(Car car) {
        this.car = car;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public ArrayList<Service> getServices() {
        return services;
    }

    public void setServices(ArrayList<Service> services) {
        this.services = services;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }
}

File: main/java/ru/carwash/models/Order.kt
package ru.carwash.models

import android.os.Parcel
import android.os.Parcelable
import java.util.*

/* Заказ */
data class Order(var status: Int) : Parcelable {
    var carWash: CarWash? = null
    var car: Car? = null
    var data // TODO
            : String? = null
    var time // TODO
            : String? = null
    var services // список услуг в заказе
            : ArrayList<Service>? = null
    var price = 0.0

    constructor(parcel: Parcel) : this(parcel.readInt()) {
        car = parcel.readParcelable(Car::class.java.classLoader)
        data = parcel.readString()
        time = parcel.readString()
        price = parcel.readDouble()
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeInt(status)
        parcel.writeParcelable(car, flags)
        parcel.writeString(data)
        parcel.writeString(time)
        parcel.writeDouble(price)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Order> {
        override fun createFromParcel(parcel: Parcel): Order {
            return Order(parcel)
        }

        override fun newArray(size: Int): Array<Order?> {
            return arrayOfNulls(size)
        }

        @JvmField
        var CANCELED_STATUS = 0
        @JvmField
        var ACCEPTED_STATUS = 1
        @JvmField
        var COMPLETED_STATUS = 2
    }
}